### PR TITLE
Deleted unnecessary test case files

### PR DESCRIPTION
**Pull request by tushartk**

### CODE REVIEW

File: algorithms/appo/tests/test_appo_utils.py [hunk 1]
```diff
@@ -1,23 +1,7 @@
...
-                self.assertEqual(cores, [i % cpu_count])
-            elif i == 40:
-                self.assertEqual(cores, [0, 1, 2, 3, 4])
-            elif i == 41:
-                self.assertEqual(cores, [5, 6, 7, 8, 9])
-            elif i == 42:
-                self.assertEqual(cores, [10, 11, 12, 13, 14])
-            elif i == 43:
-                self.assertEqual(cores, [15, 16, 17, 18, 19])
+        pass
```
Comment (alex-petrenko) [inline algorithms/appo/tests/test_appo_utils.py:6]: Don't delete this. I think this was a legit test.

File: utils/tests/test_utils.py [hunk 2]
```diff
@@ -1,10 +1,23 @@
...
-
-from utils.network import is_udp_port_available
+from utils.utils import cores_for_worker_process
 
 
 class TestNetwork(TestCase):
-    def test_udp(self):
-        is_udp_port_available(50301)
+    def test_cpu_affinity(self):
+        num_workers = 44
```
Comment (alex-petrenko) [inline utils/tests/test_utils.py:7]: This does not make sense. Why would you remove the is_udp_port_available line and add the rest of the code instead?
I think everything was okay and you shouldn't have changed it

File: algorithms/appo/tests/test_appo_utils.py (hunk 1 above)
Comment (tushartk) [reply algorithms/appo/tests/test_appo_utils.py:6]: I've shifted this to test_utils

File: utils/tests/test_utils.py (hunk 2 above)
Comment (tushartk) [reply utils/tests/test_utils.py:7]: The cores_for_worker_process method was in the utils folder and hence I shifted it here

**Pull request merged**